fastly/alerts_test.go

```diff
@@ -23,14 +23,26 @@ func TestClient_FastlyAlerts(t *testing.T) {
 		Dimensions:         testDimensions,
 		EvaluationStrategy: testEvaluationStrategy,
 		IntegrationIDs:     []string{},
-		Metric:             ToPointer("status_5xx"),
+		Metric:             ToPointer("status_5xx_rate"),
```

Comment:
Actually that doesn't throw an error but it should be status_5xx_rate according to https://www.fastly.com/documentation/reference/api/observability/alerts/definition/.